recipes/requirements-detector/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "requirements-detector" %}
+{% set version = "0.5.1" %}
+{% set sha256 = "37a4e31ce45dc7d7659dbe611846e0710847fb6f6ffa1944394f6044e6f81a97" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+    - astroid >=1.4
+
+test:
+  imports:
+    - requirements_detector
+
+about:
+  home: https://github.com/landscapeio/requirements-detector
+  license: MIT
```

Comment:
Doing in PR ( https://github.com/conda-forge/staged-recipes/pull/1315 ) pre-conversion.